util/path.go

```diff
@@ -0,0 +1,34 @@
+package util
+
+import (
+	"os"
+)
+
+// PathExists returns true of a path exists, false if it does not
```

Comment:
Typo: "of" is used instead of "if" in comment. (It's just a comment, but nice to fix early.)